src/components/Modals/Send/views/Details.tsx

```diff
@@ -32,14 +34,26 @@ import { SendFormFields, SendRoutes } from '../SendCommon'
 import { SendMaxButton } from '../SendMaxButton/SendMaxButton'
 
 export const Details = () => {
-  const { control } = useFormContext<SendInput>()
+  const { control, setValue } = useFormContext<SendInput>()
   const history = useHistory()
   const translate = useTranslate()
 
   const { asset, cryptoAmount, cryptoSymbol, fiatAmount, fiatSymbol, amountFieldError } = useWatch({
     control,
   })
 
+  const onAccountChange = useCallback(
+    (accountId: string) => setValue(SendFormFields.AccountId, accountId),
```

Comment:
```suggestion
    (accountId: AccountId) => setValue(SendFormFields.AccountId, accountId),
```

https://github.com/shapeshift/web/blob/6fb8179ad688faea8fd6b9c4f6b5addedd98487c/src/components/AccountDropdown/AccountDropdown.tsx#L44